src/main/java/com/zpi/token/domain/authorizationRequest/response/Response.java

```diff
@@ -1,22 +1,21 @@
 package com.zpi.token.domain.authorizationRequest.response;
 
 import com.zpi.token.domain.authorizationRequest.request.Request;
-import lombok.Builder;
 import lombok.Getter;
 
 import java.util.UUID;
 
 @Getter
 public class Response {
-    private final String code;
+    private final String ticket;
     private final String state;
 
     public Response(Request request) {
```

Comment:
remove public constructor, replace it with static constructor that provides description that it also generates ticket

Reply:
- [ ] fix